requirements.txt

```diff
@@ -1,5 +1,6 @@
 xmltodict==0.9.2
 boto>=2.36.0
```

Comment:
Can `boto` be removed?  I don't imagine we want to use both `boto` and `boto3` in the same repo.

Edit:  Oh.  I see that there's `storage3.py` that live next to `storage.py`.  So, I guess this was intentional.